operator-pipeline-images/operatorcert/static_tests/common/bundle.py

```diff
@@ -167,6 +167,26 @@ def check_bundle_release_config(bundle: Bundle) -> Iterator[CheckResult]:
             )
 
 
+def check_validate_schema_ci_config(
```

Comment:
The ci.yaml file is associated with the operator. Please move this check under the operator test suite.